x-pack/plugins/security_solution/public/timelines/components/side_panel/host_details/index.tsx

```diff
@@ -82,6 +83,8 @@ export const HostDetailsPanel: React.FC<HostDetailsProps> = React.memo(
           <ExpandableHostDetailsPageLink hostName={hostName} />
           <EuiSpacer size="m" />
           <ExpandableHostDetails contextID={contextID} scopeId={scopeId} hostName={hostName} />
+          <EuiSpacer size="m" />
+          <AssetCriticalitySelector entity={{ type: 'host', name: hostName }} />
```

Comment:
Still not sure if this is the correct placement? @SourinPaul

Reply:
After discussing with Pablo, we were wondering if it also needs to show up in the user flyout in the timeline?

Reply:
> Still not sure if this is the correct placement? @SourinPaul

Can we move the assignment component before displaying host metadata? The accordion can stay collapsed.  

> After discussing with Pablo, we were wondering if it also needs to show up in the user flyout in the timeline?

In short, yes. The users should be able to assign from every instance of the user flyout, including the timeline. Afaik, this user flyout component is reused everywhere in our app for consistency. I may be incorrect, so worth double checking that.

cc: @tiansivive @machadoum

Reply:
> Afaik, this user flyout component is reused everywhere in our app for consistency. I may be incorrect, so worth double checking that.

Not a direct answer, but calling out that the timeline view does _not_ support the new expanded flyouts. This means that, today, even when we have the new users flyout enabled, the timeline view still shows the old flyout.

Reply:
@jaredburgettelastic The key here is that the timeline does not use *flyout*s, hence the discrepancy. Perhaps it should?

Reply:
> timeline does not use flyouts

Apologies for the lack of clarity, to make this more clear:

> This means that, today, even when we have the new users flyout enabled, the timeline view still shows the old ~~flyout~~ panel, which is the exact same underlying component as the old flyout, but embedded directly into the timeline view.

Reply:
> The key here is that the timeline does not use flyouts

Apologies for the lack of clarity, to make this more clear:

> This means that, today, even when we have the new users flyout enabled, the timeline view still shows ~~the old flyout~~ a panel containing much of the same components used by the old host/user flyout, but embedded directly into the timeline view.